docs/specifications/1-introduction.md

```diff
@@ -0,0 +1,172 @@
+implement support immediately), a new RFC should be created in the
+[`awslabs/aws-cdk-rfcs`] repository, following the [RFC Process]. This ensures
+enough time is spent considering alternatives to breaking changes, and to create
+consensus that the change is desirable before time is spent implementing it.
+
+[`awslabs/aws-cdk-rfcs`]: https://github.com/awslabs/aws-cdk-rfcs
+[RFC Process]: https://github.com/aws/aws-cdk-rfcs#what-the-process-is
+
+> While going through the RFC process upfront is **strongly recommended**,
+> contributors may choose not to file an RFC for a behavior change. In this case
+> however, any core maintainer may decide that an RFC is required and block the
+> contribution until the RFC process has been followed.
+
+When the RFC is **ready**, a GitHub pull request is created that must contain:
+
+- Relevant additions or modifications to the specification documents
+- Relevant additions or modifications to the compliance suite
+- Implementation of the new behavior, including new or updated tests in all the
+  language bindings
+
+The pull request's body must reference the RFC if there has been one, and
+otherwise must include all discussion necessary to explain the reasoning behind
+the proposal (including alternatives considered, risks, ...).
+
+#### <a id="addressing-gaps"/> :mag: Addressing Gaps
+
+Proposal that increase the specification's coverage (desribing behavior that
+already exists) are handled as GitHub pull request that must contain the
+following elements:
+
+- Relevant additions to the specification documents
+- New compliance test(s) that enshrine the described behavior
+- Implementation of the new compliance test(s) for all *Generally Available*
+  language bindings
+
+The pull request body should provide pointers to any and all elements that can
+be used to verify that the behavior that is described is indeed what is
+currently implemented.
+
+#### <a id="trivial"/> :thumbsup: Trivial Changes
+
+Proposal of trivial changes, such as correcting typos in the document, or
+re-phrasing elements of the specification without altering the meaning
+(typically to improve clarity) are handled in a simple GitHub pull request.
+
+
+## Design Tenets (unless you know better ones)
+
+* *jsii* APIs strive to feel idiomatic in all supported languages.
+* *jsii* applications behave identically regardless of the language they are
+  written in. It favors correctness over performance.
+* *jsii* **does not** attempt to support all TypeScript idioms (many features of
+  TypeScript cannot be expressed in some target languages).
+  * When prohibiting an idiom, *jsii* strives to provide an error message that
+    gives the user insight into why the pattern cannot be supported.
+* *jsii* does not force API design opinions on the developer:
+  * Reserved names are limited to a minimum.
+  * TypeScript API design patterns that will not transpose well in other
```

Comment:
What is the meaning of 'transpose well'? Tenets should make it clear when there will be an error and when there will be a warning.